tensorboard/plugins/mesh/summary_v2.py

```diff
@@ -87,22 +76,22 @@ def _get_json_config(config_dict):
   return json_config
 
 
-def mesh(name, vertices, faces=None, colors=None, display_name=None,
-       description=None, config_dict=None, step=None):
+def mesh(name, vertices, faces=None, colors=None, description=None,
+    config_dict=None, step=None):
   """Writes a TensorFlow mesh summary.
 
   Args:
-    name: A name for this summary operation.
+    name: Uniquely identifiable name of the summary op. Could be replaced by
+      combination of name and type to make it unique even outside of this
+      summary.
     vertices: Tensor of shape `[dim_1, ..., dim_n, 3]` representing the 3D
       coordinates of vertices.
     faces: Tensor of shape `[dim_1, ..., dim_n, 3]` containing indices of
       vertices within each triangle.
     colors: Tensor of shape `[dim_1, ..., dim_n, 3]` containing colors for each
       vertex.
-    display_name: If set, will be used as the display name in TensorBoard.
-      Defaults to `name`.
-    description: A longform readable description of the summary data. Markdown
-      is supported.
+    description: Optional long-form description for this summary, as a
```

Comment:
As mentioned can we make `description` always the last keyword argument, to be consistent with other summary APIs?  (Both in the actual signature and here in the docstring.)

Ditto for mesh_pb() as well.

Reply:
You got it. I guess it only applies to public interfaces, right? (I didn't change _write_summary private method).